src/components/ProductTours/tests/ProductTours.test.jsx

```diff
@@ -55,26 +55,33 @@ const ToursWithContext = ({
   canManageLearnerCredit = false,
 }) => (
   <Router history={historyMock(pathname)}>
-    <Provider store={store}>
-      <EnterpriseSubsidiesContext.Provider value={{
-        canManageLearnerCredit,
-      }}
-      >
-        <SubsidyRequestsContext.Provider value={{
-          subsidyRequestConfiguration: {
-            subsidyType,
-            subsidyRequestsEnabled,
-          },
-        }}
-        >
-          <>
-            <ProductTours />
-            <p id={TOUR_TARGETS.LEARNER_CREDIT}>Learner Credit Management</p>
-            <p id={TOUR_TARGETS.SETTINGS_SIDEBAR}>Settings</p>
-          </>
-        </SubsidyRequestsContext.Provider>
-      </EnterpriseSubsidiesContext.Provider>
-    </Provider>
+    <Route
+      path={`/${ENTERPRISE_SLUG}/admin/:page`}
+      render={
+        () => (
+          <Provider store={store}>
```

Comment:
nit: Should the `Provider` be rendered above the `Route` and/or `Router`? I think that would more closely match how the component tree is laid out in the application.